python/lsst/jointcal/jointcal.py

```diff
@@ -668,12 +678,26 @@ def _fit_astrometry(self, dataName=None):
 
         return Astrometry(fit, model, sky_to_tan_projection)
 
+    def check_stars(self):
+        """Count measured and reference stars per ccd and warn/log them."""
+        for ccdImage in self.associations.getCcdImageList():
+            stars = ccdImage.countStars()
```

Comment:
I strongly suggest unpacking the returned value into `nMeasStars` and `nRefStars` or similar. I think it would make the code clearer, because the name `stars` sounds like a collection of star objects, not a count or pair of counts, and `stars` gives no hint as to which component is which.